src/dotnet/UI.Blazor/Components/Button/HeaderButton.razor

```diff
@@ -0,0 +1,36 @@
+@namespace ActualChat.UI.Blazor.Components
+
+<button @ref="ButtonRef"
```

Comment:
Why do we need new component for Button? What is missing in the existent Button or RoundButton component?